go.mod

```diff
@@ -14,29 +14,53 @@ require (
 	github.com/labstack/echo/v4 v4.11.4
 	github.com/mitchellh/copystructure v1.2.0
 	github.com/mitchellh/mapstructure v1.5.0
+	github.com/openziti/sdk-golang v0.22.21
 	github.com/rcrowley/go-metrics v0.0.0-20201227073835-cf1acfcdf475
+	github.com/sirupsen/logrus v1.9.3
```

Comment:
These two new dependencies are new to EdgeX and need to go thru the vetting process and TSC approval.
See process here: https://wiki.edgexfoundry.org/display/FA/Vetting+Process+for+3rd+Party+Dependencies

Reply:
I've updated the PR but i dunno how to encourage the -dependency and -security-review labels to be added to the PR.

Reply:
Added